docs/FAQ.md

```diff
@@ -58,7 +58,7 @@ Remember this is a free/open community support. We may not be available 24/7 to
 
 - **Question 4**: When using LightGBM GPU, I cannot reproduce results over several runs.
 
-- **Solution 4**: It is a normal issue, there is nothing we/you can do about. For reproducible results, please use CPU version.
+- **Solution 4**: It is a normal issue, there is nothing we/you can do about, you may try to use `gpu_use_dp = true` for reproducibility. You may also use CPU version.
```

Comment:
I think you can add the link of that comment here https://github.com/Microsoft/LightGBM/pull/560#issuecomment-304561654.